datahub-graphql-api/src/main/resources/gms.graphql

```diff
@@ -0,0 +1,367 @@
+scalar Long
```

Comment:
A comment for why this is needed?

Reply:
Long is not a default GQL type, we are introducing a type 'extension' here

Reply:
Will add